test/e2e/specs/editor/blocks/spacer.spec.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.describe( 'Spacer', () => {
+	test.beforeEach( async ( { admin } ) => {
+		await admin.createNewPost();
+	} );
+
+	test( 'can be created by typing "/spacer"', async ( { editor, page } ) => {
+		// Create a spacer with the slash block shortcut.
+		await page.click( 'role=button[name="Add default block"i]' );
+		await page.keyboard.type( '/spacer' );
+		await page.keyboard.press( 'Enter' );
+
+		expect( await editor.getEditedPostContent() ).toMatchSnapshot();
+	} );
+
+	test( 'can be resized using the drag handle and remains selected after being dragged', async ( {
+		page,
+		editor,
+	} ) => {
+		// Create a spacer with the slash block shortcut.
+		await page.click( 'role=button[name="Add default block"i]' );
+		await page.keyboard.type( '/spacer' );
+		await page.keyboard.press( 'Enter' );
+
+		const resizableHandle = page.locator(
+			'[aria-label="Block: Spacer"] .block-library-spacer__resize-container .components-resizable-box__handle'
+		);
+		await editor.dragAndResize( resizableHandle, { x: 0, y: 50 } );
+		expect( await editor.getEditedPostContent() ).toMatchSnapshot();
+
+		const selectedSpacer = page.locator(
+			'[data-type="core/spacer"].is-selected'
+		);
+		expect( selectedSpacer ).not.toBe( null );
```

Comment:
Instead of asserting "selected", I think maybe the `focused` state makes more sense?

```suggestion
		await expect( page.locator( 'role=document[name="Block Spacer"i]' ) ).toBeFocused();
```

Reply:
@kevin940726 Thank you for your review!
I have made some fixed as you have reviewed,
but `toBeFocused` could not work at my environment.
It seemed that the block still was focused, but the '.toBeFocused' test was failed. 😢

Reply:
I was wondering if maybe this test needs to check if the resized spacer block has the `is-selected` class?

Reply:
Hmmm 🤔, I wonder why `toBeFocused` fails? I'm not sure if `is-selected` is a public API or not, but I don't think it makes much sense to assert that it exists. As a last resort, we can use the gutenberg selector API to check if the block is being selected.

Reply:
I'm sorry, I was just copying and pasting the code you wrote, and didn't realize that I just had the wrong selector name for the block 😅
`[name="Block Spacer"i]` => `[name="Block: Spacer"i]`
　(@t-hamano Thank you for telling it!)

So, I fixed it. Plz review again. 🙏🏻
@kevin940726